### PR TITLE
fix release script

### PR DESCRIPTION
**Pull request by jessfraz**
merge it ;)

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by tianon** [comment]
Can you please expound on why this change makes sense?  I'm not seeing any possible way for this change to actually fix anything unless one of those args intentionally has spaces for some bizarre reason. :wink:

**Comment by jfrazelle** [comment]
its the * glob

**Comment by jfrazelle** [comment]
DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb

then it had quotes around it and exploded

**Comment by tianon** [comment]
but removing the quotes here shouldn't actually fix that -- that'd have to happen on the earlier line

**Comment by jfrazelle** [comment]
oh but it doessss i literally coud not do the release without this

File: hack/release.sh [hunk 1]
```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```
Comment (tianon) [inline hack/release.sh]: Quotes are intentionally not on this line specifically so that the glob expands. :confused:

**Comment by jfrazelle** [comment]
start here https://botbot.me/freenode/docker-maintainers/2015-05-28/?msg=40320639&page=1

File: hack/release.sh (hunk 1 above)
Comment (tianon) [inline hack/release.sh]: Ugh, this is actually because `something=` doesn't do globs like it should.  If you change this line to the following, it should work:
```bash
DEBFILE=( bundles/$VERSION/ubuntu/lxc-docker*.deb )
```

(and then `$DEBFILE` will get the first one, just like `$BASH_SOURCE` gets the first item in the `BASH_SOURCE` array)

Comment (jfrazelle) [inline hack/release.sh]: but whats wrong with just no quotes haha

Comment (tianon) [inline hack/release.sh]: We'll just get another `shellcheck` drive-by that tries to add them back, and we'll forget this whole conversation. :wink:

Comment (tianon) [inline hack/release.sh]: and it's taking advantage of a side effect instead of explicitly using a glob

Comment (jfrazelle) [inline hack/release.sh]: ok okkkk ill do the other thing

**Comment by tianon** [comment]
LGTM, but it's worth running a test push to a random S3 bucket to be sure :smile:

**Comment by jfrazelle** [comment]
ya im trying now

**Comment by jfrazelle** [comment]
can confirm it works, tested on a random bucket :)
